main.py

```diff
@@ -44,6 +44,8 @@
 current_channel: Optional[TextChannel] = None
 # The media in the current channel
 current_channel_content: Optional[List] = None
+# The local filepaths of media from the current bust
+current_bust_content: Optional[List] = None
```

Comment:
for deeper type inspection.

```suggestion
current_bust_content: Optional[List[str]] = None
```